.eslintrc.js

```diff
@@ -86,6 +86,7 @@ module.exports = {
         'no-magic-numbers': 'off',
         '@typescript-eslint/no-var-requires': 'off',
         '@typescript-eslint/no-non-null-assertion': 'off',
+        '@typescript-eslint/no-floating-promises': 'off',
```

Comment:
jest will warn, that a sync call to act should not be awaited. So I turned this rule off in tests